svc-api/router/router.go

```diff
@@ -219,6 +220,7 @@ func Router() *iris.Application {
 	systems.Get("/{id}/Storage/{rid}", system.GetSystemResource)
 	systems.Get("/{id}/Storage/{rid}/Drives", system.GetSystemResource)
```

Comment:
create a route party as storage under this add these uri

Reply:
Addressed the comment